smithy-model/src/main/resources/software/amazon/smithy/model/loader/prelude.smithy

```diff
@@ -1032,3 +1032,62 @@ list LocalMixinTraitList {
             must target a valid trait.""")
 @private
 string LocalMixinTrait
+
+/// Defines the priority-ordered list of compression algorithms supported by the
+/// service operation.
+@private
+list RequestCompressionEncodingsList {
+    /// Encodings are NOT constrained in a Smithy enum since
```

Comment:
This is a documentation comment, but it should probably be a regular comment with this having it's own actual documentation.